api/v1beta1/cluster_types.go

```diff
@@ -72,6 +72,24 @@ type ClusterSpec struct {
 	// this feature is highly experimental, and parts of it might still be not implemented.
 	// +optional
 	Topology *Topology `json:"topology,omitempty"`
+
+	// availabilityGates specifies additional conditions to include when evaluating Cluster availability.
+	// A Cluster is available if:
+	// * Cluster's `RemoteConnectionProbe` and `TopologyReconciled` conditions are true and
+	// * the control plane `Available` condition is true and
+	// * all worker resource's `Available` conditions are true and
+	// * all conditions defined in AvailabilityGates are true as well.
```

Comment:
Writing this with bullets looks good here, but if you look at the generated CRD description, it won't render well when folks use `kubectl explain` on the field. Best to use complete sentences rather than bullets for this kind of thing.

If you are fixed on bullets, try adding extra blank lines between each bullet, I believe that may help rendering

Reply:
How does it look like with kubectl? The CRD yaml looks okay

Reply:
Hmm, maybe they've fixed the controller-tools generator, so we should double check, but, typically content like this would render without new lines and look something like
```
availabilityGates specifies additional conditions to include when evaluating Cluster availability. A Cluster is available if:* Cluster's `RemoteConnectionProbe` and `TopologyReconciled` conditions are true and* the control plane `Available` condition is true and* all worker resource's `Available` conditions are true and* all conditions defined in AvailabilityGates are true as well.
```

Reply:
I think I might remember a change in CT a while ago

Reply:
@JoelSpeed I think it was this one https://github.com/kubernetes-sigs/controller-tools/pull/870/files (CT v0.14.0)

Reply:
I have simplified the definition of AvailableGates, dropping the duplicated definition of the Available condition (so now we have this only in one place, reducing the risk of misalignment and confusion in future.

Reply:
Ahh nice! Then I will no longer point this out to folks, thanks for checking